app/src/main/res/values/strings.xml

```diff
@@ -461,6 +461,12 @@
     <!-- Title for the customize home screen section with Pocket. -->
     <string name="customize_toggle_pocket">Pocket</string>
 
+    <!-- Wallpapers -->
+    <!-- Title for preference to open wallpapers -->
+    <string name="customize_wallpapers">Wallpapers</string>
+    <!-- Content description for various wallpapers. The first parameter is the name of the wallpaper -->
+    <string name="content_description_wallpaper_name">Wallpaper Item: %1$s</string>
```

Comment:
Can you add an appropriate string prefix for this? Since it's grouped under `<!-- Wallpapers -->`, you will find in our `strings.xml` that we tend to align towards using a prefix to indicate strings that are within the same screen. However, this might mean `customize_wallpapers ` should probably just be under `<!-- Customize Home -->` and used as the title. You will have to look and see what makes sense.